web-client/src/presenter/computeds/AdvancedSearch/practitionerSearchHelper.test.ts

```diff
@@ -130,6 +146,7 @@ describe('practitionerSearchHelper', () => {
       },
     });
     expect(result).toMatchObject({
+      numOfResultsDisplay: '1',
```

Comment:
Since the issue was numbers with commas, the test if it exists should look at a number with a comma in it.